src/main/scala/io/iohk/ethereum/vm/DataWord.scala

```diff
@@ -79,10 +88,56 @@ class DataWord private (private val n: BigInt) extends Ordered[DataWord] {
 
   def *(that: DataWord): DataWord = DataWord(this.n * that.n)
 
-  def /(that: DataWord): DataWord = DataWord(this.n / that.n)
+  def /(that: DataWord): DataWord = zeroCheck(that) { DataWord(this.n / that.n) }
```

Comment:
When you see a `/` method on a numeric type you would expect it to mean the usual division. It's not the case so it would make sense add a comment. But I think it's even better to rename it to `div`.